CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
```

Comment:
This is a major release, right? We need to add the new version number here.

Reply:
I think the final release number should now be 1.0.0